dev/how-to/maintain/identify-abnormal-queries/identify-slow-queries.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: 慢查询日志
 category: how-to
-aliases: ['/docs-cn/sql/slow-query/']
+aliases: ['/docs-cn/sql/slow-query/','/docs-cn/dev/how-to/maintain/identify-slow-queries/']
```

Comment:
Please align this change in https://github.com/pingcap/docs/pull/1804/ @toutdesuite

Reply:
Files are updated. PTAL. @yikeke